### DISCUSSION TITLE
Notebook power/performance management

### DISCUSSION THREAD

**Discussion by volfpeter** [elementary/streams#35] [Ideas]
Odin might change the picture, but I feel this is a pain point even in Juno/Hera.
Which application do you recommend for this purpose (TLP, Tuxedo Control Center, something else)? What would be a good way to set up and manage these settings? ...

**Comment by danirabbit** [comment]
I think this is outside the scope of these streams. I don't think we want to encourage people to take on hacks and tweaks

The author of the new GNOME power profiles daemon talks about some of the drawbacks of tools like TLP here: http://www.hadess.net/2020/09/power-profiles-daemon-new-project.html

**Reply by volfpeter** [reply]
My point is that people use these tools anyway to improve battery life (the creation of the linked project also supports this), because this is what's available and recommended on forums - hence the need for an official solution/recommendation. I understand your reasoning as a product owner. Also, I agree this might be out of scope for these streams, although nothing was written about what kind of content is in scope. But as a user, I would be interested in this topic.
